codesign/codesign.go

```diff
@@ -194,27 +194,27 @@ func SelectConnectionCredentials(authType AuthType, conn *devportalservice.Apple
 }
 
 func (m *Manager) selectCodeSigningStrategy(credentials appleauth.Credentials) (codeSigningStrategy, string, error) {
-	const manualProfilesReason = "Using Bitrise-managed code signing via API key, as Automatically managed signing is disabled in Xcode for the project."
+	const manualProfilesReason = "Using Bitrise-managed code signing assets with API key because Automatically managed signing is disabled in Xcode for the project."
 
 	if credentials.AppleID != nil {
-		return codeSigningBitriseAppleID, "Using Bitrise-managed code signing via Apple ID, as Apple ID is not supported by Xcode-managed code signing.", nil
+		return codeSigningBitriseAppleID, "Using Bitrise-managed code signing assets with Apple ID because Apple ID authentication is not supported by xcodebuild.", nil
 	}
 
 	if credentials.APIKey == nil {
-		panic("No Apple authentication credentials found.")
+		panic("No App Store Connect API authentication credentials found.")
 	}
 
 	if !m.opts.ShouldConsiderXcodeSigning {
 		return codeSigningBitriseAPIKey, "", nil
 	}
 
 	if m.opts.XcodeMajorVersion < 13 {
-		return codeSigningBitriseAPIKey, "Using Bitrise-managed code signing via API key, as Xcode-managed code signing requires at least Xcode 13.", nil
+		return codeSigningBitriseAPIKey, "Using Bitrise-managed code signing assets with API key because 'xcodebuild -allowProvisioningUpdates' with API authentication requires Xcode 13 or higher.", nil
```

Comment:
```suggestion
		return codeSigningBitriseAPIKey, "Using Bitrise-managed code signing assets with API key because 'xcodebuild -allowProvisioningUpdates' requires Xcode 13 or higher.", nil
```

Reply:
@godrei isn't `-allowProvisioningUpdates` available in Xcode 12, but only supports the Apple account configured through the UI?

Reply:
Yes that's right.